pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool.poetry]
+name = "csm-api-client"
+version = "1.0.0"
+description = "Python client library for CSM APIs"
+authors = [
+    "Ryan Haasken <ryan.haasken@hpe.com>",
+    "Eli Kamin <eli.kamin@hpe.com>",
+    "Jack Stanek <jack.stanek@hpe.com>",
+    "Pat Tovo <pat.tovo@hpe.com"
+]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/Cray-HPE/sat"
+packages = [{include = "csm_api_client"}]
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
Can you point me at the docs for this `^` syntax, please?

Reply:
See the docs here: https://python-poetry.org/docs/dependency-specification/#caret-requirements
>Caret requirements allow [SemVer](https://semver.org/) compatible updates to a specified version. An update is allowed if the new version number does not modify the left-most non-zero digit in the major, minor, patch grouping. For instance, if we previously ran poetry add requests@^2.13.0 and wanted to update the library and ran poetry update requests, poetry would update us to version 2.14.0 if it was available, but would not update us to 3.0.0. If instead we had specified the version string as ^0.1.13, poetry would update to 0.1.14 but not 0.2.0. 0.0.x is not considered compatible with any other version.

This is equivalent to `python >= 3.8, < 4.0`.